lego/apps/companies/serializers.py

```diff
@@ -36,6 +38,11 @@ def create(self, validated_data):
         return super().create(validated_data)
 
 
+class StudentCompanyContactSerializer(BasisModelSerializer):
+    class Meta:
+        model = StudentCompanyContact
+        fields = ("id", "company_id", "student_id", "semester_id")
```

Comment:
Is the semester_id supposed to be the year? (In the example it's "1"?

Reply:
It's the ID of the Semester object